python/torch_mlir/dialects/torch/importer/jit_ir/CMakeLists.txt

```diff
@@ -7,7 +7,14 @@ include(TorchMLIRPyTorch)
 set(Torch_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../../../../../libtorch/share/cmake/Torch")
 find_package(Torch 1.11 REQUIRED)
 
-TorchMLIRConfigureLibTorch()
+option(USE_LOCAL_PYTORCH "Build from local PyTorch in environment" ON)
```

Comment:
maybe call this TORCH_MLIR_USE_INSTALLED_PYTORCH

Reply:
done